docs/Contributing.adoc

```diff
@@ -1,17 +1,40 @@
-== Contributing 
+== Contributing
 
 If you're reading this, hopefully you are considering helping out with the Ansible Agnostic Deployer for Red Hat enablement.
 
 Herein lies the contribution guidelines for helping out with this project. Do take the guidelines here literally, if you find issue with any of them or you see room for improvement, please let us know via a GitHub issue.
 
-=== Rules
+=== General rules
 
 * The Ansible <<coc,Code of Conduct>> still applies.
 * For git messages, branch names, ..., follow <<gitstyle,Git Style Guide>>
 * Should you wish to work on a completely new standard, GREAT, but please file an issue for tracking.
 * To contribute, fork and make a pull request against the `development` branch.
 * All tasks should be in YAML literal.
 
+=== About reviewing Pull Requests
+
+. Do not merge a PR after reviewing unless explicitely asked for
+** Approval and merging are different.
+** Other people might be reviewing at the same time, or want to review that PR
+** The PR might not be fully tested by the author yet
+. If a specific person was requested for review, don't merge before that person reviewed, or before the request was canceled.
+. Take your time. If your PR is merged tomorrow instead of today, is it a big deal ?
```

Comment:
Pace of merging should be on commiter's better judgement of impact of the PR :slightly_smiling_face:
IMHO it might get to be a big deal to slow down the developement too much - people will usually rather correct mistakes, then get annoyed by too long waiting.

Reply:
OK fair enough, removing that line.

Reply:
I really don't think this line needs to be removed. This whole thing is not a black and white set of rules, although there are some. It is also to remind people to be thorough when they are doing commits, reviews, and merges. Very very very few PRs *need* to be merged today vs tomorrow. If they do, you can make that quite clear in the PR or reach out to someone directly for a more prompt review.

Reply:
Yes, i just tried to capture the essence of what we discussed, but happy to rephrase if necessary.

Let me know with requestchanges / approval

Reply:
I think we should merge, this is _just_ a doc update and we can iterate on it if we feel it needs further refinement over time.